src/utils.js

```diff
@@ -187,3 +187,24 @@ export function parseJwt (token) {
   var base64 = base64Url.replace('-', '+').replace('_', '/');
   return JSON.parse(window.atob(base64));
 }
+
+export function getSpawnPositionInCircle(originX, originZ, radius, rotation) {
+  let x = originX + radius * Math.cos(rotation);
+  let z = originZ + radius * Math.sin(rotation);
+
+  return {x: x, z: z};
+}
+
+export function getRotationToTarget(eye, target) {
+  let m4 = new THREE.Matrix4();
```

Comment:
Since this function only gets called once it doesn't matter here, but just FYI if this was getting called more than once you would want to re-use the threejs primitives like: https://github.com/mozilla/mr-social-client/blob/b08841cdaaa800b0ff96b1f5ccd3b453314f886d/src/components/character-controller.js#L199-L211